keel-web/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -68,93 +69,51 @@ class ApplicationService(
    * for the application and reindexing the data so that it matches the right format.
    *
    * This function assumes there's a single delivery config associated with the application.
-   *
-   * The algorithm is as follows:
-   *
-   * For each environment in the delivery config:
-   * |    For each artifact in the environment:
-   * |    |    Build a map of artifact versions by state (e.g. pending, current, previous, etc.).
-   * |    |    For each pair of (state, versions):
-   * |    |    |    For each version in this state:
-   * |    |    |    |    Get and cache an [ArtifactSummaryInEnvironment] for this version in this environment.
-   * |    |    |    |    Create and cache the [ArtifactVersionSummary] for this version.
-   * |    |    Finally, create the [ArtifactSummary] by looking up the "inner" summaries that were built above.
-   * Return a flat list of all the artifact summaries for all environments in the delivery config.
```

Comment:
Is this no longer right?

Reply:
Nope, algorithm is much simpler now, so I thought it was unnecessary to describe in pseudo-code.